tests/e2e/script_test.go

```diff
@@ -325,16 +325,12 @@ func patchObjectWithExternallyManagedFields(h *create.Harness, obj *unstructured
 }
 
 func setAnnotation(h *create.Harness, obj *unstructured.Unstructured, k, v string) {
-	patch := &unstructured.Unstructured{}
```

Comment:
Ah yeah, I bet this is causing havoc because the owner is the same - good catch !